cpp/tests/io/parquet_test.cpp

```diff
@@ -3348,7 +3348,7 @@ TEST_F(ParquetWriterTest, CheckPageRows)
   // check first page header and make sure it has only 5000 values
   int fd = open(filepath.c_str(), O_RDONLY);
   unsigned char buf[1024];
-  read(fd, buf, sizeof(buf));
+  (void)!read(fd, buf, sizeof(buf));
```

Comment:
I think it is better to use `[[maybe_unused]]` instead of the old casting style, if it works.

Reply:
Should we verify the number of bytes read actually match what we expected? Instead of trying to sidestep the return value, why don't we actually use it?

Reply:
First, I tried `[[maybe_unused]]`. It didn't work.
[cppreference ](https://en.cppreference.com/w/cpp/language/attributes/maybe_unused)says, `This attribute can appear in the declaration of the following entities`
So, it can be used only in _declaration_.

Reply:
Right usage is [`std::ignore`](https://en.cppreference.com/w/cpp/utility/tuple/ignore)

Reply:
Right usage is [`std::ignore`](https://en.cppreference.com/w/cpp/utility/tuple/ignore)